hcpannot/io.py

```diff
@@ -929,23 +929,26 @@ def load_report(region, rater, sid, h, reports_path=None):
     except Exception as e:
         pass
     return data
+
 def load_allreports(region,
                     reports_path=None,
                     include_mean=True,
-                    sids=subject_list):
+                    sids=None): 
```

Comment:
`sids=None` is a good way to set up the argument, but there needs to be an `if` statement inside the function:

```python
if sids is None:
    sids = subject_list
```